go.mod

```
@@ -49,6 +49,8 @@ require (
 	pgregory.net/rapid v1.1.0
 )
 
+require github.com/Ethernal-Tech/blockchain-event-tracker v0.0.0-20231202204931-b886edca635a
```

Append to the previous require block.

```suggestion
	pgregory.net/rapid v1.1.0
    github.com/Ethernal-Tech/blockchain-event-tracker v0.0.0-20231202204931-b886edca635a
)
```

---

Fix: https://github.com/Ethernal-Tech/blade/pull/41/commits/a02c0d2a235db4149700e29029a7da3623661b5a